sdk/nodejs/output.ts

```diff
@@ -534,6 +534,20 @@ export function secret<T>(val: Input<T | undefined>): Output<Unwrap<T | undefine
         o.isKnown, Promise.resolve(true), o.allResources!());
 }
 
+/**
+ * [unsecret] behaves the same as [output] except the returned output takes the existing output and unwraps the secret
+ */
+export function unsecret<T>(val: Output<T>): Output<T> {
+   return new Output(
```

Comment:
I think we want to first do `const o = output(val);` to ensure we're operating on an instance of `Output`, which also makes it safe to call `.allResources` per the comment in the implementation of `secret`:

https://github.com/pulumi/pulumi/blob/bd1554d33e45f73c8543bc79252f50bb83a03a1d/sdk/nodejs/output.ts#L526-L535